cmd/example-tests/main.go

```diff
@@ -53,6 +54,13 @@ func main() {
 		panic(fmt.Sprintf("couldn't build extension test specs from ginkgo: %+v", err.Error()))
 	}
 
+	// Environment selector information can be added to test specs to support filtering by environment
+	specs.Walk(func(spec *et.ExtensionTestSpec) {
+		if spec.Name == "[sig-testing] openshift-tests-extension should support test-skips via environment flags" {
+			spec.Include(et.PlatformEquals("aws"))
+		}
+	})
```

Comment:
```
specs.Select(NameContains("openshift-tests-extension should support test-skips via environment flags")).Include(et.PlatformEquals("aws"))
```

https://issues.redhat.com/browse/TRT-1920

Reply:
I will work on that card next, in a follow up PR.